algorithms/matrix/matrix chain recurssion.py

```diff
@@ -0,0 +1,17 @@
+from sys import maxint
+import time
+def matrix(p,i,j):
```

Comment:
please use more more descriptive function name.
I don't know what this function is for